### ISSUE TITLE
Incomplete imagesdir support

### ISSUE THREAD

**Issue by slonopotamus** [asciidoctor/asciidoctor-intellij-plugin#395]
Given the following `diagram.adoc` file:

----
```asciidoc
= Diagrams
:imagesdir: a

[ditaa, "a"]
....
+--+
|A |
+--+
....

:imagesdir: b

[ditaa, "b"]
....
+--+
|B |
+--+
....

:imagesdir: c

[ditaa, "c"]
....
+--+
|C |
+--+
....

:imagesdir: d
```
----

I expect all three images to be shown in preview.

In fact, only the last image is shown, first two are empty:

![image](https://user-images.githubusercontent.com/92637/73344708-e2bec400-4293-11ea-8f40-fc8357b36d8f.png)

`asciidoctor -D output/ -r asciidoctor-diagram diagram.adoc` properly handles this file, so this is asciidoctor-intellij-plugin specific problem:

![image](https://user-images.githubusercontent.com/92637/73345030-6082cf80-4294-11ea-80d5-b492d3eb34df.png)

AsciiDoc Plugin Version: 0.30.31

IDE Name and Version: IntelliJ IDEA 2019.3.1 (Ultimate Edition) Build #IU-193.5662.53, built on December 18, 2019

**Comment by ahus1** [comment]
Thank you for providing a minimal example to reproduce it. 

**TL;DR:** The pre-release 0.30.35 supports this example as long as your diagram names are distinct. 

As an alternative to multiple imagesdir definitions I'd recommend to add the path to the filename of the diagram like `[ditaa, "a/a"]`

Some background and history: I've changed the behavior how to display preview images as part of #345 to improve support of imagesdir containing ".." that was traversing outside of the temporary preview image folder. 

I've defined imagesoutdir with the temporary image folder. This lead to a solution where only one definition of imagesdir per document was supported in the preview, and duplicate diagram names would overwrite each other. This was first released as part of 0.30.31 (first pre-release was 0.30.25). 

In this ticket I'm changing it again to support the case you describe with multiple definitions of imagesdir inside the same document. You'll still run into problems when two diagrams have the same name as the preview will create all diagrams in the same folder as the plugin specifies imagesoutdir to direct the output of the images to a separate temporary folder.

A new pre-release 0.30.35 of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/CONTRIBUTING.adoc#eap).

**Comment by ahus1** [comment]
A new pre-release 0.30.36 of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/CONTRIBUTING.adoc#eap).
